### PR TITLE
[FIX] Fix report to coveralls.io

### LINKED ISSUE #52: Coverage result provided by coveralls.io is wrong
result by coveralls.io: 40%
result in the build output: 86%

### PR DESCRIPTION
**Pull request by lmignon**
try to fix #52

### CODE REVIEW

**Comment by lmignon** [comment]
coverage result is now correctly computed by coveralls.io https://coveralls.io/builds/3156165

**Comment by petrus-v** [comment]
I'm fine with that, letting time to @jssuzanne to develop it's plugging to ignore import on coverage as suggest in #52 

@jssuzanne I wonder if your plugging works with coveralls?

@lmignon, isn't it over due to run coverage with nose and coveralls.io ?

**Comment by lmignon** [comment]
@petrus-v We don't need a plugin here. Coveralls only collect data generated by nosestests --with-coverage. When trying to fix the problem, I have observed that the data transmitted to coveralls.io was not the same with or without the coverage lib installed.

**Comment by petrus-v** [comment]
@lmignon,

oh I see, sorry about my quick answer, and my miss understanding! 

I thought coverall was launch twice, which was wrong
```
$ coveralls
Submitting coverage to coveralls.io...
Coverage submitted!
Job #29.1
```

And find out that what you have done in ``.coveragerc`` let config as need.

I don't remember well @jssuzanne purpose maybe it's a bit different use case.

I gonna merge it.

File: .travis.yml [hunk 1]
```diff
@@ -16,14 +16,15 @@ virtualenv:
 install:
   - pip install -e .[dev]
   - pip install coveralls
+  - pip install coverage
```
Comment (bwrsandman) [inline .travis.yml]: I thought coveralls brought in coverage...

**Comment by lmignon** [comment]
@bwrsandman me too but without this line the reported result to coveralls is wrong. I'm not proud of this fix but at least we now have the correct result.

**Comment by bwrsandman** [comment]
That's strange, because the output is 
```
Requirement already satisfied (use --upgrade to upgrade): coverage in /home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages
```

**Pull request merged**